### DISCUSSION TITLE
is it possible to somehow add conditions for updates of versions?

### DISCUSSION THREAD

**Discussion by jan-mrm** [renovatebot/renovate#18767] [Request Help] [answered]
Hey there, is it possible to add additional conditions to *renovate bot* that have to be matched in order to suggest version updates?

For example: I was thinking about some sort of staging of dependency updates. I was wondering if it is possible to have renovate suggest updates for the first of my (lets say) two stages/environments. And only suggest updates for the second stage configuration when the version is actually present in the first stage, by for example providing a custom script to check if the version is present in some folder / certain files of my configuration repository. Some sort of hook before an update is suggested?

**Comment by rarkins** [comment, marked as answer]
No, that's not possible

**Reply by jan-mrm** [reply]
Thanks for your really quick answer btw.
I just stumbled upon your issue https://github.com/renovatebot/renovate/issues/18609, I guess that could work if one really wanted it 😃
